bedrock/mozorg/redirects.py

```diff
@@ -229,9 +229,6 @@ def decider(request, **kwargs):
     # bug 854561
     redirect(r'^projects/mozilla-based(\.html|/)?', 'mozorg.projects.mozilla-based'),
 
-    # bug 851727
-    redirect(r'^projects/powered-by(\.html|/)?', 'mozorg.powered-by'),
```

Comment:
I think we probably still need this redirect but should change the destination. And we should add a new redirect for `/about/powered-by` that goes to the same place. I'm not sure what a good destination might be, though. Maybe just the home page?

Reply:
@craigcook I will redirect to the homepage for now. I'm sure a bug will be filed if that creates issues contextually within the site. 🙂